lib/jwt_sessions/store_adapters/abstract_store_adapter.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'redis'
+
+module JWTSessions
+  module StoreAdapters
+    class AbstractStoreAdapter
+      class << self
+        def instance_by_symbol(adapter, options = {})
```

Comment:
Maybe, `instance_by_name`? We can use strings too, not only symbols

Reply:
👍